scripts/hash-exec-copy.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
As a general comment I'd say that this script might only make sense in `spells-mainnet`. Not opposed to have it here either, but won't be used.

Reply:
yeah, opened here for discussion, once approved I could port it to `spells-mainnet` and close the Draft PR.

Reply:
Hmm if will be just for mainnet, I guess we should do the draft PR there. Anyway not a big deal, just wondering if we also want the script here.

Reply:
yeah, we could keep parity for the script and have it here as well, as you think it's best.

Reply:
Yeah fine for me. Let's keep it here as well.